testsuite/docker/kerberos/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:22.04
```

Comment:
@mstruk Are you fine with this being based on Ubuntu? Strimzi IMHO does not use Ubuntu images anywhere else. So I would feel better if this used Red Hat UBI images as all our other projects. But I can live with this if you are fine with it.

Reply:
Now that you pointed this out, if I remember correctly, we had some Travis build issues with Ubuntu images on some architectures.

@akaczano Could you try use:
`FROM registry.access.redhat.com/ubi8/ubi`

or

`FROM registry.access.redhat.com/ubi8/openjdk-17`

You may need to install some additional packages.

Reply:
Sure; I'll give that a try.

Reply:
I took a look at this and it looks very challenging to set it up on `ubi8`. Tried a few leads with no luck. The needed package `krb5-server` doesn't seem to be available in the repo or is behind some kind of subscription wall. The setup is apparently very complicated as it is, and putting it together with any alternative would be just as complicated if not more. I suggest we keep it as is, and exclude the test run on architectures that will cause problems due to unavailability of platform specific container images.